config/routes.rb

```diff
@@ -89,81 +88,6 @@ def put(*_args); end
       resources :volumes,            :only => [:index, :show]
     end
 
-    namespace :v0x1, :path => "v0.1" do
```

Comment:
Maybe not for this version, but in the future (v1.0+) we should probably return a `HTTP 410 Gone`